manifest.k8s.yml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ecrreverseproxy
+data:
+  PORT: "{PORT}"
+  REGION: {REGION}
+  REGISTRY: {REGISTRY}
+  ECR_REGISTRY: {ECR_REGISTRY}
+
+---
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: ecrreverseproxy
+  name: ecrreverseproxy
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ecrreverseproxy
+  template:
+    metadata:
+      labels:
+        app: ecrreverseproxy
+    spec:
+      containers:
+        - image: marjamis/ecr_reverse_proxy:latest
+          name: ecrreverseproxy
+          ports:
+          - containerPort: {PORT}
+            protocol: TCP
+          envFrom:
+          - configMapRef:
+              name: ecrreverseproxy
+      volumes:
+        - name: certs
+          secret:
+            secretName: ecrreverseproxy
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ecrreverseproxy
+  name: ecrreverseproxy
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-internal: "0.0.0.0/0"
+  namespace: kube-system
+spec:
+  type: LoadBalancer
```

Comment:
This should be purely a ClusterIP as this will be used via an Ingress Controller.

Reply:
Also look into restricting outside access completely as this should really only need to be used internally to the cluster.